artview/components/tools.py

```diff
@@ -105,6 +105,7 @@ def onPick(self, event):
         '''Get value at the point selected by mouse click.'''
         xdata = event.xdata # get event x location
         ydata = event.ydata # get event y location
+        #XXX this fails if not inside plot region
```

Comment:
yes it does, at least for me. I needed some time to track down the reason, but I couldn’t come up with a simple solution, so I just marked it.